programs/server/Server.cpp

```diff
@@ -1810,6 +1813,8 @@ try
                                                                      &CurrentMetrics::MaxDDLEntryID, &CurrentMetrics::MaxPushedDDLEntryID));
         }
 
+        ProfileEvents::increment(ProfileEvents::ServerStartupMilliseconds, startup_watch.elapsedMilliseconds());
```

Comment:
may just a little lower - after these lines?
https://github.com/ClickHouse/ClickHouse/blob/a0fe26f5069dd425f3862a3eec4cca6066c7a7d5/programs/server/Server.cpp#L1826-L1828

Reply:
addressed it